### DISCUSSION TITLE
[Projects Beta] Personal Views

### DISCUSSION THREAD

**Discussion by yousseftelda** [community/community#6423] [Projects and Issues]
It would be nice to allow members to create views that are private to them to avoid cluttering the views. A use case would be a personal view with a filter to only show issues assigned to themselves.

**Comment by Oliquae** [comment]
You can actually can do this while using the filter feature, like so : 
![image](https://user-images.githubusercontent.com/6728594/137750843-1e625df6-7120-4dbc-acfe-1fcf94ad855f.png)

**Reply by nicholasconfer** [reply]
The issue here is they would have to create a new view for each person, and then all those views would clutter up the board.

**Reply by andybergon** [reply]
For this specific issue you can use a filter like `assignee:@me` but the general suggestion would still be useful to let people play with private views they own.

**Comment by rnreekez** [comment]
Optionally, the ability to filter on "assigned to me" which would have a different context depending on the authenticated user would be nice.

**Reply by jasoncavett** [reply]
This is an excellent idea. We have quite a list of cluttered views - especially as our team is growing.

**Reply by rnreekez** [reply]
It happened! @me is now supported as a filter for assignee! Thanks GH

**Reply by jasoncavett** [reply]
Well that's awesome! You just made my day - I didn't realize that had come out.

**Reply by ezzygemini** [reply]
![So Happy](https://media.giphy.com/media/i21tixUQEE7TEqwmYa/giphy.gif)

**Reply by thefringeninja** [reply]
Doesn't work for `reviewers` though... 🤔

**Comment by ezzygemini** [comment]
+1   Make this happen!!!

**Comment by nkolev92** [comment]
Worth noting that while `assignee:@me ` is helpful for some cases, it doesn't really help in scenarios like stand-up, where 1 person is screen sharing and going through every person on the call.

**Reply by ezzygemini** [reply]
Nice try JIRA

**Reply by rnreekez** [reply]
In that scenario, I would think you'd want a table view grouped by assignees.

**Comment by lanormerh** [comment]
This is a quite basic feature. Please implement this

**Comment by Julian-Dumitrascu** [comment]
I'd like to discuss choosing whom we let sense the issues in a project view and create issues that will be displayed in that view.
Where would you like us to discuss this? Examples:
a. here
You extend this conversation to managing access to project views. Such a feature has the application you want.
b. I request this feature in a new conversation.

**Comment by turadg** [comment]
Until this is implemented, you can have personal views in your own bookmarks.

Whenever you modify an existing view, the changes go into the URL until you save it. You can simply bookmark that URL instead of saving it.

One feature of this is that your bookmark is of customizations to the underlying view. If that saved view changes, yours will too.

**Comment by danemacmillan** [comment]
This would be fantastic for those of us who use one Project to streamline issues across many repositories and provide a base set of Views for day-to-day movement forward, but where some PMs and other stakeholders would want to manipulate the Views a bit without piling on the Views and making it less organized. They could always create a new Project, sure, but then they also need to associate all the issues with the new project, which is time-consuming for many repositories.

Generally, it would be nice to associate permissions with Views, so that different teams could see different Views, instead of, again, either choosing to have way too many Views or choosing to add certain issues to other Projects for them.

**Comment by zsloane** [comment]
This would be incredibly useful. Right now everyone wants to look at things in a different way which leads to way too many views for everyone and it quickly becomes cluttered and overwhelming

**Comment by CharlieThomas** [comment]
I have found the @me feature really useful but I think it still needs a personal/private option as people want to organise their board differently to be most effective, finding a layout that suits everyone's way of working is challenging

**Comment by tomas-forsman** [comment]
Personal views would be very apprechiated. Our boards view has exploded with custom tabs.

**Comment by ramasheshasai** [comment]
Hi there,

That’s a great idea — having personal or private views would definitely help reduce clutter in shared spaces. A common use case (like you mentioned) would be a personal filter that only shows issues assigned to yourself, without affecting what the rest of the team sees.

Right now, GitHub doesn’t support private views, but a workaround is to use saved searches or filters in your own browser (e.g., bookmark a query like is:open assignee:@me). It’s not as convenient as native private views, but it gives you a quick way to jump back to your own issues.

Might be worth submitting this as a feature request through the GitHub feedback form so it gets visibility with the product team.

**Comment by linde-ibis** [comment]
I agree. The @me is good, but everyone wants to sort and lay out their view differently.

**Comment by HaraldWikeroy** [comment]
Establish a new project and load all your issues into it. Then you can do what you want in there?
